src/BaGet.Core/BaGet.Core.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="$(MicrosoftExtensionsPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(MicrosoftExtensionsPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(MicrosoftExtensionsPackageVersion)" />
-    <PackageReference Include="NuGet.Packaging" Version="$(NuGetPackageVersion)" />
+    <PackageReference Include="NuGet.Packaging" Version="5.3.1" />
```

Comment:
Please revert this change. You can bump up the NuGet.Packaging version here:

https://github.com/loic-sharma/BaGet/blob/05bfc61e1ff86517d2e4571f12524275c2355a0d/src/Directory.Build.props#L33